app/code/Magento/SalesInventory/Model/Plugin/Order/Validation/OrderRefundCreationArguments.php

```diff
@@ -18,19 +18,13 @@
  */
 class OrderRefundCreationArguments
 {
-    /**
-     * @var ReturnValidator
-     */
-    private $returnValidator;
-
     /**
      * OrderRefundCreationArguments constructor.
      * @param ReturnValidator $returnValidator
      */
     public function __construct(
-        ReturnValidator $returnValidator
+        private readonly ReturnValidator $returnValidator
```

Comment:
A trailing comma makes future additions easier to review.
```suggestion
        private readonly ReturnValidator $returnValidator,
```